sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -192,16 +198,22 @@ export class ApplicationStudentsController extends BaseController {
       payload.data.selectedOfferingEndDate = studyEndDate;
     }
 
-    const submissionResult = await this.formService.dryRunSubmission(
-      programYear.formName,
-      payload.data,
-    );
+    const submissionResult: DryRunSubmissionResult<ApplicationData> =
```

Comment:
Thank you for doing this change.
Can the `dryRunSubmission` be also adapted as below to complete the change?
```ts
async dryRunSubmission<T = any>(
    formName: string,
    data: unknown,
  ): Promise<DryRunSubmissionResult<T>>
```

Reply:
Yes. Thanks!